client/modules/accounts/templates/profile/profile.js

```diff
@@ -2,14 +2,19 @@ import * as Collections from "/lib/collections";
 import { Meteor } from "meteor/meteor";
 import { Template } from "meteor/templating";
 
+let allShops = [];
+let addShop = false;
+
 /**
  * onCreated: Account Profile View
  */
 Template.accountProfile.onCreated(() => {
   const template = Template.instance();
-
+  Meteor.call("shopNames/all", (err, result) => {
+    return err ? err : (allShops = result.vendorShopNames);
```

Comment:
Es6 arrow body style will avoid the use of return statement here. Why not try:
Meteor.call("shopNames/all", (err, result) => (err ? err : (allShops = result.vendorShopNames)));